lib/DDG/Spice/CodeDay.pm

```diff
@@ -0,0 +1,56 @@
+package DDG::Spice::CodeDay;
+# ABSTRACT: Get all CodeDay events, sorted by closest to farthest
+
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 0;
+
+name "CodeDay";
+source "CodeDay.org";
+icon_url "https://codeday.org/assets/img/favicon.ico";
+description "Get information about a CodeDay or find the nearest one.";
+primary_example_queries "codeday near me", "when is codeday?", "codeday san diego";
+secondary_example_queries "codeday", "codeday seattle", "what is codeday bay area?";
+category "programming";
+topics "programming", "geek", "computing", "entertainment";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/CodeDay.pm";
+attribution github => ["tjhorner", "TJ Horner"],
+            twitter => "tjhorner";
+
+spice to => 'https://clear.codeday.org/api/regions/$1?term=$2&lat=$3&lng=$4&limit=50';
```

Comment:
> The third solution is to adjust your API and this code so that we can always just return an array of defined data, and don't need to worry about passing a blank space as a value.

This sounds like the best solution. I'll add an endpoint for this later today.